src/main/java/mcscheduler/model/ModelManager.java

```diff
@@ -123,18 +122,21 @@ public void setWorker(Worker target, Worker editedWorker) {
         mcScheduler.setWorker(target, editedWorker);
     }
     @Override
-    public float calculateWorkerPay(Worker worker) {
+    public int calculateWorkerShiftsAssigned(Worker worker) {
         Integer numberOfShiftsAssigned = 0;
         ObservableList<Assignment> assignments = getFullAssignmentList();
         for (Assignment assignment : assignments) {
+            if (assignment.getRole() instanceof Leave) {
```

Comment:
Perhaps will be better to use `Leave.isLeave`? Technically new Role("Leave") is also a leave but is not a `Leave` object (even though this is disallowed from happening by the factory method, just in case).
```suggestion
            if (Leave.isLeave(assignment.getRole())) {
```

Reply:
Noted thanks for the suggestion